RyftUI/Source/Controllers/RyftDropInPaymentViewController.swift

```diff
@@ -198,15 +204,21 @@ public final class RyftDropInPaymentViewController: UIViewController {
     }
 
     public func handleRequiredAction(
-        returnUrl: String,
+        returnUrl: URL?,
```

Comment:
might be worth updating the PaymentSession model `returnUrl` to a URL type too, or update the example here in the README to transform the string to a URL: https://github.com/RyftPay/ryft-ios#implementing-the-ryftdropinpaymentdelegate

Reply:
yeah good spot, I'll have a look at decoding the JSON on the `PaymentSession` so it's already a `URL` in the next change